boards/admin.py

```diff
@@ -4,3 +4,8 @@
 from django.contrib import admin
 
 # Register your models here.
+from django.contrib import admin
+from .models import Board
+
+admin.site.register(Board)
+
```

Comment:
Maybe, you haven't reach that yet, but you can also configure your own admin classes. I.e.:

```
class BoardAdmin(admin.AdminModel):
    list_display = [.....] -> display them in the list view, on the admin
    list_editable = [.....] -> for the ones displayed, the ones on this list will be editable from the list view
    list_filter = [.....] -> creates a filter on the side panel with occurrences. Useful for booleans and choice fields
    search_fields = [....]  -> creates a search bar and you will be able to search for the fields on this list. I.e put `name` and look for a name you know, it will filter results based on that
    ......

admin.site.register(Board, BoardAdmin)
```

This is actually very easy to use and extremely useful. Try including Board attributes on the `list_display`, etc.

Like:
`list_display = ["name", "description]`

etc

You can also build admins for the rest of the models! Don't forget to register them! 😄